src/unix/utilsunx.cpp

```diff
@@ -1088,6 +1088,11 @@ bool wxIsPlatform64Bit()
 #endif
 }
 
+wxString wxGetCpuArchitectureName() {
+    const wxString machine = wxGetCommandOutput(wxT("uname -m"));
+    return machine;
```

Comment:
```suggestion
    return wxGetCommandOutput(wxT("uname -m"));
```
Also very minor, of course, but what's the point of this temporary variable?